packages/react/src/themes/teams/components/Segment/segmentVariables.ts

```diff
@@ -11,19 +11,32 @@ export type SegmentColorSchemeMapping = TeamsSchemeMappingWithAreas<
 export interface SegmentVariables {
   colorScheme: SegmentColorSchemeMapping
   color: string
+  colorDisabled: string
```

Comment:
Should be `disabledColor`: `Slot => Prop => Name`

Reply:
done thanks

Reply:
Still the same 😢 

![image](https://user-images.githubusercontent.com/14183168/60005740-644fd300-966f-11e9-8283-8ddb6930ca58.png)

Please check that variables were renamed